### DISCUSSION TITLE
Datepicker as feature?

### DISCUSSION THREAD

**Discussion by scherler** [plouc/nivo#1673] [Ideas]
Hi there,

I just finished a standalone datepicker component based on bootstrap and it is basically the timeRange component as a combo with an input field and overlay. 

>  datepicker allows the user to select a specific date or a range of dates. Follows aria guidelines from w3.org - Date Picker Dialog Example for keyboard navigation. React component based on bootstrap, typescript and SVG, build with webpack.

https://github.com/scherler/datepicker is the repo. Is there an interest to move it into a package here? Is that something that sparkles interest?

**Comment by wyze** [comment]
No thank you. We are keeping this as a dataviz library.
